src/app/code/community/Omise/Gateway/Model/OmiseCharge.php

```diff
@@ -17,7 +19,9 @@ public function createOmiseCharge($params)
 
     /**
      * Capture a charge that retrieve from charge id
-     * @param array $params
+     *
+     * @param  array $params
```

Comment:
@guzzilar The difference of parameter name in PHPDoc and source code.

PHPDoc is `$params`

Source code is `$id`